docs/built-in-models/training.md

```diff
@@ -0,0 +1,48 @@
+# Training on Encrypted Data
+
+Concrete ML offers the possibility to train [SGD Logistic Regression](../developer-guide/api/concrete.ml.sklearn.linear_model.md#class-sgdclassifier) on encrypted data. The [logistic regression training](../advanced_examples/LogisticRegressionTraining.ipynb) example shows this feature in action.
+
+This example shows how to instantiate a logistic regression model that trains on encrypted data:
+
+```python
+parameters_range = (-1.0, 1.0)
+
+sgd_clf_binary_simulate = SGDClassifier(
```

Comment:
not sure why this model is named `sgd_clf_binary_simulate` . I'm mostly taking about this "simulate" part, but I think you can just name it `model` or `model_encrypted_fit` for simplicity

Reply:
Agreed let's make codeblock very user friendly.